targets/repository.proj

```diff
@@ -64,19 +98,42 @@
                             SourcePath="$(SourceBuiltPackagesPath)" />
```

Comment:
Shouldn't we disable the mutation of the nuget.config with this change?

Reply:
We are.  The condition on this target is `Condition="'$(NuGetConfigFile)' != ''"`.  And we are not setting that property when the repo supports the SourceOverrideRepoApi.

https://github.com/dotnet/source-build/pull/219/files#diff-fcefba7596e1fe07d9907df477c1384fR31

```xml
<PropertyGroup Condition="'$(SourceOverrideRepoApiImplemented)' != 'true'">
      <NuGetConfigFile Condition="'$(NuGetConfigFile)' == '' and Exists('$(ProjectDirectory)NuGet.config')">$(ProjectDirectory)NuGet.config</NuGetConfigFile>
```

Reply:
cool